frontend/packages/console-shared/src/components/synced-editor/index.tsx

```diff
@@ -144,12 +146,13 @@ type SyncedEditorProps = {
     formContext: { [key: string]: any };
     yamlContext: { [key: string]: any };
   };
-  FormEditor: React.FC<any>;
+  FormEditor: React.ElementType<any>;
```

Comment:
```suggestion
  FormEditor: React.ComponentType;
```

Reply:
thanks @christianvogt , I'm commiting these changes except I keep the generics `<any>` (necessary else props is assumed as `{}` and generates TS error).

Reply:
@jotak ok! using `any` is fine for now since that's what we have previously. We really should type both `FormEditor` and `YAMLEditor` props correctly though because we do pass in specific props to these components.

Reply:
hold on, last commit amended!